tests/cfServicesUtil.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
Maybe it is better to disable this rule only in places where it is relevant but not for the whole file?